frontend/components/DocumentEditToolbar/DocumentEditToolbar.jsx

```diff
@@ -111,7 +117,7 @@ export default class DocumentListToolbar extends Component {
         <div class={styles['padded-group']}>
           <ButtonWithOptions
             accent="save"
-            disabled={disabled}
+            disabled={hasValidationErrors || disabled}
```

Comment:
My idea to use `disabled` was to keep it generic so that parents can choose to disable the controls in the toolbar for other reasons if they wish to. We're using `hasConnectionIssues` in `DocumentEdit` and being explicit about that, I don't think the toolbar necessarily needs to be aware of the concept of connection issues. It just enables or disables the controls based on what the parent tells it.

Your call, though.